crmsh/utils.py

```diff
@@ -2292,4 +2149,255 @@ def interface_choice():
     # should consider interface format like "ethx@xxx"
     interface_list = re.findall(r'(?:[0-9]+:) (.*?)(?=: |@.*?: )', out)
     return [nic for nic in interface_list if nic != "lo"]
+
+
+class IP(object):
+    """
+    Class to get some properties of IP address
+    """
+
+    def __init__(self, addr):
+        """
+        Init function
+        """
+        self.addr = addr
+
+    @property
+    def ip_address(self):
+        """
+        Create ipaddress instance
+        """
+        return ipaddress.ip_address(self.addr)
+
+    @property
+    def version(self):
+        """
+        Get IP address version
+        """
+        return self.ip_address.version
+
+    @classmethod
+    def is_mcast(cls, addr):
+        """
+        Check whether the address is multicast address
+        """
+        cls_inst = cls(addr)
+        return cls_inst.ip_address.is_multicast
+
+    @classmethod
+    def is_ipv6(cls, addr):
+        """
+        Check whether the address is IPV6 address
+        """
+        return cls(addr).version == 6
+
+    @classmethod
+    def is_valid_IP(cls, addr):
```

Comment:
Why `IP` in capital letters here? In other parts of the code we use `ip`

Reply:
OK, changed